pkg/cmd/dev/vulcan.go

```diff
@@ -10,8 +10,15 @@ import (
 	"go.uber.org/zap"
 )
 
-func vulcan(f *cmdutil.Factory, cmd *DevCmd) error {
-	command := vul.Command("", "dev")
+func vulcan(f *cmdutil.Factory, cmd *DevCmd, isFirewall bool) error {
+
+	var command string
+
+	if isFirewall {
```

Comment:
You could create a variable that has the command to be run and only use one "if" instead of "if - else". 
If isFirewall, you append. Then you do vul.Command ("",  varCommand). 
Similar to vulcanParams in buildCommand